pages/Authentication/logout.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="https://shuttle-ui.netlify.app/Components/main.css">   
+    <link rel="stylesheet" href="authentication.css">
+    <link rel="icon" href="../../resource/asset/thumbnail.png">
+    <title>Login-Closet</title>
+</head>
+<body>
+    <!-- Navigation -->
+    <header>
+        <nav class="navigation">
+            <div class="nav-brand-logo">
+                <img class="nav-img" src="../../resource/asset/_Logo.png">
```

Comment:
Fix this too

Reply:
Thankyou for your comment. I have made the changes